### PR TITLE
stylize student list (#63)

### PR DESCRIPTION
**Pull request by michal367**
stylize student list
obraz

### CODE REVIEW

**Review by rozchlastywacz — APPROVED** [review]
Kinda smelly code, but I will approve that this time

File: src/lecturer/components/studentListView/StudentListHead.tsx [hunk 1]
```diff
@@ -27,9 +28,31 @@ export function StudentListHead(props: StudentListHeadProps) {
...
+            margin: "15px auto"
+        },
+        head: {
+            padding: "20px 15px",
+            textAlign: "left",
+            fontWeight: 500,
+            color: "#fff",
+            
+            backgroundColor: theme.palette.primary.main,
+            "& span": {
```
Comment (Petros9) [inline src/lecturer/components/studentListView/StudentListHead.tsx:45]: What does it mean? ("& span")?

**Review by Derstilon — APPROVED** [review]
Approved

File: src/lecturer/components/studentListView/StudentListHead.tsx (hunk 1 above)
Comment (michal367) [reply src/lecturer/components/studentListView/StudentListHead.tsx:45]: It's selector pointing to upper selector

For example:
head: {
    width: "200px",
    "& td": {
        color: "red"
    }
}

in CSS means:
.head{
    width: 200px;
}
.head td{
    color: red;
}

Comment (Petros9) [reply src/lecturer/components/studentListView/StudentListHead.tsx:45]: thanks

**Pull request merged**